webviews/components/merge.tsx

```diff
@@ -65,10 +65,10 @@ const StatusChecks = ({ pr }: { pr: PullRequest }) => {
 					<StateIcon state={status.state} />
 					<div className="status-item-detail-text">
 						<span>{getSummaryLabel(status.statuses)}</span>
-						<a href="javascript:void(0)" aria-role="button" onClick={toggleDetails}>
-							{showDetails ? 'Hide' : 'Show'}
-						</a>
 					</div>
+					<button className='secondary' onClick={toggleDetails} style={{ padding: '0 5px', marginLeft: 'auto' }}>
```

Comment:
Let's avoid inline styles. What would be great here would be to extract the styles out of the header's new button group into something like `.small-button` so we can reuse them across both areas.